setup.py

```diff
@@ -0,0 +1,14 @@
+import versioneer
+from setuptools import setup, find_packages
+
+setup(name='pmgr',
+      version=versioneer.get_version(),
+      cmdclass=versioneer.get_cmdclass(),
+      license='BSD',
+      author='SLAC National Accelerator Laboratories',
+      packages=find_packages(),
+      include_package_data=True,
+      description=('Parameter Manager for LCLS Device Configurations'),
+      entry_points={'console_scripts': ['pmgr = pmgr.pmgr:main',
```

Comment:
What does `entry_points={console_scripts:  ...}` do as opposed to just `scripts=[...]`?

Reply:
`entry_points={console_scripts: ...}` automatically generates a script that calls a function. `scripts` includes things that are marked as executables. In this case, the existing `pmgr`, etc. scripts were fully setting up a conda environment from a fixed location on NFS, which no longer makes sense if this is itself a conda package.